### DISCUSSION TITLE
Mantık yürütme sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#282] [Zeka Sorusu]
Sadece verilen öncüllerden sonuca kesin ulaşıldığı durumlara EVET, aksi takdirde HAYIR olarak yanıtlayınız.
1. Çıkarım
Öncül: Tüttürülen her şey sağlığa faydalıdır.
Öncül: Sigara tüttürülür.
Sonuç: Sigara sağlığa faydalıdır.
2. Çıkarım
Öncül: Dört ayaklı bütün hayvanlar tehlikelidir.
Öncül: Minyatür kanişler tehlikeli değildir.
Sonuç: Minyatür kanişler dört ayaklı hayvan değildir.
3. Çıkarım
Öncül: Bütün işsizler yoksuldur.
Öncül: Ali işsiz değildir.
Sonuç: Ali yoksul değildir.
Hangi çıkarımlar mantıksal olarak EVET hangileri HAYIR.

**Comment by HabilCcabbar** [comment]
1. Çıkarım: **EVET**
 Sigara sağlığa zararlı olmasına rağmen verilen öncüllerden bu sonuca ulaşılır.
2. Çıkarım: **HAYIR**
 Bu çıkarım yapılamaz çünkü bize minyatür kanişin bir hayvan olduğu verilen öncüllerde söylenmiyor. Ama eğer bu önemli değilse cevap EVET'tir.
3. Çıkarım: **HAYIR**
 Bu da yanlıştır çünkü bütün işsizler yoksulken bütün yoksullar işsiz değildir.
